### PR TITLE
Docker all

### PR DESCRIPTION
**Pull request by Joaodemeirelles**
Adding Dockerfile and docker-compose to create container with functioning pipeline

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM continuumio/miniconda3
+
+COPY . /descartes-rpa/
```
Comment (teslajoy) [inline Dockerfile]: change  naming convention from descartes-rpa to descartes_rpa (in all code)

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+
+  descartes_rpa:
+    build:
+      context: .
+    volumes:
+      - /home/joao/gsoc/descartes-rpa/data/output:/descartes_rpa/data/output
```
Comment (teslajoy) [inline docker-compose.yml]: pass in users path as an environment variable for compatibility

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+FROM continuumio/miniconda3
```
Comment (teslajoy) [inline Dockerfile:1]: we may need to change this, but for now I think this is great

Comment (Joaodemeirelles) [reply Dockerfile:1]: OK! I agree hahaha

File: Dockerfile (hunk 1 above)
Comment (Joaodemeirelles) [reply Dockerfile]: Do we change the repo name as well? I've already changed the rest

Comment (teslajoy) [reply Dockerfile]: Yes! @SolomonShorser-OICR already changed the repo name : )

Comment (Joaodemeirelles) [reply Dockerfile]: Thanks @teslajoy and @SolomonShorser-OICR :D

File: docker-compose.yml (hunk 2 above)
Comment (Joaodemeirelles) [reply docker-compose.yml]: Now changed fullpath to relative path of data folder

**Comment by teslajoy** [comment]
Looks good! go ahead and merge : )

**Pull request merged**